supersim.go

```diff
@@ -25,41 +31,54 @@ var genesisL1JSON []byte
 var genesisL2JSON []byte
 
 var DefaultConfig = Config{
-	l1Chain: anvil.Config{ChainId: 1, Port: 8545, Genesis: genesisL1JSON},
-	l2Chains: []anvil.Config{
-		{ChainId: 10, Port: 9545, Genesis: genesisL2JSON},
-		{ChainId: 30, Port: 9555, Genesis: genesisL2JSON},
-	},
+	l1Chain:  Chain{anvilConfig: anvil.Config{ChainId: 1, Port: 8545, Genesis: genesisL1JSON}, opSimConfig: op_simulator.Config{Port: 8546}},
+	l2Chains: []Chain{{anvilConfig: anvil.Config{ChainId: 10, Port: 9545, Genesis: genesisL2JSON}, opSimConfig: op_simulator.Config{Port: 9546}}, {anvilConfig: anvil.Config{ChainId: 30, Port: 9555, Genesis: genesisL2JSON}, opSimConfig: op_simulator.Config{Port: 9556}}},
 }
 
 type Supersim struct {
 	log log.Logger
 
-	l1Chain  *anvil.Anvil
-	l2Chains map[uint64]*anvil.Anvil
+	l1Anvil  *anvil.Anvil
```

Comment:
Nice work!

In order to simplify the changes in this file, I wonder if we should simply have OpSimulator manage the lifecycle of the the underlying chain, in this case anvil. So that we dont need reference to each individual pieces here

Not worth addressing in the PR. Just some food for thought for ways to clean up this code in the future

Reply:
Sounds good, I wont address in this PR, since I want to keep this change small. 

If we want to stick to the [original design](https://github.com/ethereum-optimism/design-docs/blob/7fce2c9e51bdd180c29d8805331905861d6c7c51/ecosystem/supersim.md#architecture), we should create an `Orchestrator` service that handles spinning up all the anvil instances and `OpSimulator` instances.